src/Plugins/OrientationAnalysis/src/OrientationAnalysis/Filters/Algorithms/FindKernelAvgMisorientations.cpp

```diff
@@ -210,7 +210,7 @@ Result<> FindKernelAvgMisorientations::operator()()
 
   ParallelData3DAlgorithm parallelAlgorithm;
   parallelAlgorithm.setRange(Range3D(0, udims[0], 0, udims[1], 0, udims[2]));
-  parallelAlgorithm.setParallelizationEnabled(true);
+  parallelAlgorithm.setParallelizationEnabled(false);
```

Comment:
Why are we turning multithreading off for certain filters?  Does multithreading and out-of-core not work well together?